### PR TITLE
Fix shell script type

### PR DESCRIPTION
**Pull request by foresmac**
Following on from #35, shell scripts typically end in `.sh`, not `.bash`. Also added a test for filetype mapping.

@jrit

### CODE REVIEW

File: source/index.js [hunk 1]
```diff
@@ -315,7 +315,7 @@ cvr.linesMissing = function ( coverage )
 cvr.getFileType = function ( filePath )
 {
     var types = {
-        bash: "bash",
+        sh: "bash",
```
Comment (jrit) [inline source/index.js]: Just do both, meaning keep the old line and add the new one

Comment (foresmac) [inline source/index.js]: easy peasy

**Comment by foresmac** [comment]
@jrit Having two extensions that map to the same type didn't work with my clever test loop; are you satisfied with how the test works now or would you prefer I modify?

**Comment by jrit** [comment]
I'm not very concerned about it, but if you just reverse the map (key=>value, value=>key) and then adjust the below tests accordingly it isn't a difficult change

**Comment by foresmac** [comment]
So obvious once it was pointed out. Thanks @jrit.

**Comment by jrit** [comment]
LGTM

**Comment by jrit** [comment]
published 1.0.2 to npm

**Pull request merged**